src/m_menu.c

```diff
@@ -3922,6 +3935,40 @@ static void M_ResetGamma(void)
   I_SetPalette(W_CacheLumpName("PLAYPAL",PU_CACHE));
 }
 
+static const char *sound_module_menu_strings[] = {
+  "Standard", "OpenAL 3D", NULL
+};
+
+static const char *sound_resampler_menu_strings[] = {
+  "Nearest", "Linear", "Cubic", NULL
+};
+
+static void M_UpdateAdvancedSoundItems(void)
+{
+  DISABLE_ITEM(snd_module != 1, gen_settings2[gen2_sndhrtf]);
+  DISABLE_ITEM(snd_module != 1, gen_settings2[gen2_sndabsorption]);
+  DISABLE_ITEM(snd_module != 1, gen_settings2[gen2_snddoppler]);
+}
+
+static void M_SetSoundModule(void)
+{
+  M_UpdateAdvancedSoundItems();
+
+  if (!I_AllowReinitSound())
+  {
+    // TODO: If extension not present, draw a warning to restart to see changes?
```

Comment:
Yes, please!

Reply:
That'd be `warn_about_changes(S_PRGWARN)`.

Reply:
Thanks! https://github.com/fabiangreffrath/woof/pull/1118/commits/db7afed09c9bb9b1fa8562b6f2de6c925de13a27